### ISSUE TITLE
Allow X notifications every Y min. Or wait Z min after answering to send the next notification.

### ISSUE THREAD

**Issue by DippedPretzels** [unalkalkan/AnkiDroid-Companion#7]
Awesome app - thanks a lot for this!  I just have a suggestion for a feature - at the screen to select a deck, add an option to only send (1-some) notifications every X min.  Either that, or the option to add a delay in min after answering a notification.

**Comment by unalkalkan** [comment]
Hey hey, thanks for the kind words! I'm glad it's useful to you.

It's an interesting use case. So correct me if I understand wrong, you don't want to be constantly exposed to notifications. You'd prefer the app to send the next notification after an interval, starting from the moment you reply your latest notification.

> the option to add a delay in min after answering a notification

In my opinion this one is a lot easier to do than the first one you've proposed, and as you said it can be behind a config that is disabled by default. I'm planning another feature close to this one, so I will take a look at this when I do that!

**Comment by DippedPretzels** [comment]
Yep, I find it hard to get a chunk of time to focus on a whole block of cards, so spreading them out would be better.  I tried to do this with the app Buzzkill, but it has trouble with Companion for some reason.

`the option to add a delay in min after answering a notification`

Whatever route seems easier to pull off.  Could be a pull down menu at the Companion start screen (0 = immediately, 5 min, 10 min, 15, 30....) or a text field for custom time.

Oddly I've found few flashcard apps with support for notifications, and only one with timed notifications (but no SRS), so I think you'd have something novel.
